elixir/apps/web/lib/web/live/actors/users/new.ex

```diff
@@ -52,16 +48,13 @@ defmodule Web.Actors.Users.New do
   def handle_event("change", %{"actor" => attrs}, socket) do
     changeset =
       attrs
-      |> map_memberships_attr()
       |> Actors.new_actor()
       |> Map.put(:action, :insert)
 
     {:noreply, assign(socket, form: to_form(changeset))}
   end
 
   def handle_event("submit", %{"actor" => attrs}, socket) do
-    attrs = map_memberships_attr(attrs)
-
```

Comment:
After new actor is created I think we should redirect not to the view page but to the one where we edit it's groups, because there is no point in having an actor that is not a member anywhere.

Reply:
I don't think I'm following.  The code is redirected to 
```
~p"/#{socket.assigns.account}/actors/users/#{actor}/new_identity"
```
after the user is created right?  

The flow when creating a new actor should now be:
* create actor
* create identity
* edit group memberships for newly created actor

Reply:
Sorry, I forgot that we already redirecting there. Will `new_identity` redirect to `edit_groups` even if you just add a new identity to an existing actor? If that's the case we might need to pass a query parameter (eg. `new_identity?next_step=manage_groups`) so that UX is not weird when you just add an identity vs create a new actor.

Reply:
Ah, yeah, good call.  Didn't think about adding a new identity to an existing actor.  Will add the param like you suggested.